rubocop-capybara.yml

```
@@ -0,0 +1,25 @@
+#
+## https://github.com/rubocop/rubocop-capybara/
+#
+
+require: rubocop-capybara
+
+# https://docs.rubocop.org/rubocop-rspec/cops_rspec_capybara.html#rspeccapybaraspecificmatcher
+Capybara/SpecificMatcher:
+  Enabled: true
+
+# https://docs.rubocop.org/rubocop-rspec/cops_rspec_capybara.html#rspeccapybaraspecificfinders
+Capybara/SpecificFinders:
+  Enabled: true
+
+# https://docs.rubocop.org/rubocop-rspec/cops_rspec_capybara.html#rspeccapybaranegationmatcher
+Capybara/NegationMatcher:
+  Enabled: true
```

```suggestion
  Enabled: true
  EnforcedStyle: have_no
```

Deberiamos de usar el have_no siempre, y no negar el have_to ya que no son equivalentes.